ansible/configs/ansible-automation-platform/roles/deploy_automationcontroller_cfg/tasks/main.yml

```diff
@@ -0,0 +1,107 @@
+---
+
+# TODO: Remove, rework after Ansible Automation Platform 2 ships
+# ansible 2.10 or higher required,
+
+- name: Remove yum/dnf installed ansible
+  dnf:
+    name: ansible
+    state: absent
+
+- name: Ensure yum/dnf pre-requisites in place
+  package:
+    name: "{{ deploy_automationcontroller_dnf_packages }}"
+    state: present
+
+- name: Install 2.11 or greater installation venv for running automation controller installer
+  include_tasks: automationcontroller_installer_venv.yml
+
+- name: Download Ansible controller installer tar.gz
+  get_url:
+    url: "{{ automationcontroller_installer.url }}"
+    dest: /tmp/automationcontroller.tar.gz
+    password: "{{ automationcontroller_installer.password }}"
+    username: "{{ automationcontroller_installer.username }}"
+
+- name: Create installer directory for Ansible controller
+  file:
+    path: /tmp/automationcontroller_installer
+    state: directory
+
+- name: Extract Ansible controller installer tar.gz
+  unarchive:
+    src: /tmp/automationcontroller.tar.gz
+    dest: /tmp/automationcontroller_installer
+    remote_src: true
+    extra_opts:
+      - '--strip-components=1'
+      - '--show-stored-names'
+
+
+# World's ugliest short term hack for fixing broken installer for clustering
+# dest: '/etc/systemd/system/automation-controller.service'
+
+# - name: Short term hack to fix Automation controller remote_src issue
+#   lineinfile:
+#     path: /tmp/automationcontroller_installer/collections/ansible_collections/ansible/automation_platform_installer/roles/automationcontroller/tasks/tasks.yml
+#     insertafter: ".*src: '/lib/systemd/system/automation-controller.service'"
+#     state: present
+#     line: "    remote_src: true"
+
+- name: Create template based inventory file for Ansible controller install
+  template:
+    src: automationcontroller_inventory.j2
+    dest: /tmp/automationcontroller_installer/inventory
+
+- name: Run the Automation Controller installer
+  #shell: "source /opt/venvs/venv_automationcontroller_install/bin/activate; ./setup.sh -e ansible_become_method=sudo -e ansible_become=True"
+  shell: "source /opt/venvs/venv_automationcontroller_install/bin/activate; ./setup.sh"
+  args:
+    chdir: /tmp/automationcontroller_installer
+  async: 1400
+  poll: 15
+  tags:
+    - deploy_automationcontroller_installer
+
+- name: Wait for Ansible controller to be up
+  uri:
+    url: "https://{{groups['automationcontroller'][0] }}/api/v2/ping/"
+    method: GET
+    user: "{{ deploy_automationcontroller_admin_user }}"
+    password: "{{ deploy_automationcontroller_admin_password }}"
+    validate_certs: false
```

Comment:
any easy way to remove `validate_certs: false` here?  like adding the cert using update-ca-trust or similar
or maybe just the `ca_path` of the uri module.  If easy, if not, it's fine.